packages/web-components/src/components/cbp-nav-item/cbp-nav-item.tsx

```diff
@@ -0,0 +1,32 @@
+import { Component, Element, Host, Prop, h } from '@stencil/core';
+import { setCSSProps } from '../../utils/utils';
+@Component({
+  tag: 'cbp-nav-item',
+  styleUrl: 'cbp-nav-item.scss',
+})
+
+export class CbpNavItem {
+
+  @Element() host: HTMLElement;
+
+  /** Specifies the context of the component as it applies to the visual design and whether it inverts when light/dark mode is toggled. Default behavior is "light-inverts" and does not have to be specified. */
+  @Prop({ reflect: true }) context: "light-inverts" | "light-always" | "dark-inverts" | "dark-always";
+
+  /** Supports adding inline styles as an object */
+  @Prop() sx: any = {};
+  
+  componentWillLoad() {
+    if (typeof this.sx == 'string') {
+      this.sx = JSON.parse(this.sx) || {};
+    }
+    setCSSProps(this.host, {
+      ...this.sx,
+    });
+  }
+
+  render() {
+    <Host>
```

Comment:
needs an onclick listener to set the item as selected/current.

Reply:
updated